hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieCompactionWriteStat.java

```diff
@@ -17,33 +17,28 @@
 package com.uber.hoodie.common.model;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-import com.uber.hoodie.common.util.FSUtils;
+
 import java.io.Serializable;
-import java.util.List;
-import java.util.Optional;
-import java.util.concurrent.atomic.AtomicLong;
-import java.util.stream.Collectors;
-import org.apache.hadoop.fs.Path;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
-public class CompactionWriteStat implements Serializable {
+public class HoodieCompactionWriteStat implements Serializable {
```

Comment:
Naming convention for everything under this package is Hoodie* so I wanted to rename this file to follow the same.